addons/mail/static/src/models/rtc/rtc.js

```diff
@@ -218,10 +213,12 @@ registerModel({
             this.videoTrack && this.videoTrack.stop();
 
             this._dataChannels = {};
-            this._fallBackTimeouts = {};
             this._outGoingCallTokens = new Set();
             this._peerConnections = {};
 
+            for (const rtcSession of this.messaging.models['RtcSession'].all()) {
+                rtcSession.update({ fallbackTimeout: clear() });
```

Comment:
This is strange (already before PR), the field is cleared, but not the actual browser timeout?

Reply:
But session `reset` is actually doing it, shouldn't this just be a call to session reset? @ThanhDodeurOdoo

Reply:
currently `_timeoutId` on `rtcSession` is the timeoutId of the debounce for `updateAndBroadcast()`, which is cleared in `reset()`.

But `_fallBackTimeouts` in `rtcSession.js` are the timeouts for the connection recovery attempts, they are properly cleared in `removePeer()`.

Reply:
Just renamed `RtcSession/fallBackTimeout` to `RtcSession/connectionRecoveryTimeout` to clarify it's related to connection recovery.